infra.bs

```diff
@@ -239,6 +239,13 @@ paths would be an insurmountable task and not productive as they would not stand
 as well as the single algorithm would. Therefore performance is best left as a field to compete
 over.
 
+<p tracking-vector>A document using the Infra Standard should not enforce specific limits on
+unconstrained inputs to allow for competition among user agents and foster ever greater data
```

Comment:
I like what Domenic suggests. And with an example 'allows for competition among user agents' part feels less strange.

Reply:
SGTM too.  Implementations need to determine how they allocate the resources necessary and so are free to set limits.

What would be really good is a little more about how specific limits are either dictated by implementation choices (which don't generally vary between installations of the same implementation and so don't contribute much to tracking), by hardware (which can vary and are consequently a source of tracking information), or both (with variable effect on tracking).  Implementations are might in a position to limit how much information is revealed through limits, such as through quantization.  This might involve sacrificing some potential performance as the applied limits might be smaller than what hardware is capable of, which puts the privacy goal in tension with any desire to maximize performance.  Again, this is an area of implementation differentiation as different approaches seek different trade-offs.

(In protocol land, there are often resource allocations that are made by others or jointly made, in which case this loose sort of approach doesn't work, but that applies on a case-by-case basis and probably isn't worth expounding upon.)